Solutions/CohesitySecurity/Tools/setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/zsh
+
+SCRIPT=$(realpath "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+cd "$SCRIPTPATH"
+
+. ./error_handler.sh
+. ./json_parser.sh
+
+. ./prerequisite.sh
+error_handler "Failed to complete prerequisites."
+
+. ./deploys.sh
+error_handler "Failed to deploy resources."
+
+. ./config.sh
+error_handler "Failed to configure resources."
+
+. ./test.sh
+error_handler "Failed to run tests."
+
```

Comment:
move this to the last.
as still great efforts to have all the test passed.

Reply:
Why not just add it and it will automatically delete when all the tests pass instead of punting it to the future? This also provides the user a way to delete the resource group from command line in case a test fails.